src/serialchemy/_tests/sample_model.py

```diff
@@ -158,3 +165,4 @@ class EmployeeSerializer(ModelSerializer):
     company_name = Field(dump_only=True)
     address = NestedModelField(Address)
     contacts = NestedModelListField(Contact)
+    marital_status = Field(serializer=EnumKeySerializer(MaritalStatus))
```

Comment:
Do this instead, to test the EnumkeyField. The serializer will get tested since it is used by the enum key field. 
```suggestion
    marital_status = EnumKeyField(MaritalStatus)
```